### PR TITLE
feat(assets): add files for m37

### PR DESCRIPTION
**Pull request by chael-mi**

### CODE REVIEW

File: src/assets/img/edition/series/2/section/2a/m37/M37_Sk1-1von1-final.svg.svg
[the remark this answers is not in the document]
Comment (musicEnfanthen) [reply src/assets/img/edition/series/2/section/2a/m37/M37_Sk1-1von1-final.svg.svg:1]: Also, remove the duplicated .svg file ending from all svg files

File: src/assets/data/edition/series/2/section/2a/m37/svg-sheets.json [hunk 1]
```diff
@@ -0,0 +1,63 @@
+{
+    "sheets": {
+        "workEditions": [],
+        "textEditions": [
+            {
+                "id": "M_37",
+                "label": "M 37",
+                "content": [
+                    {
+                        "svg": "assets/img/edition/series/2/section/2a/m37/M37_Textfassung1-1von2-final.svg",
```
Comment (musicEnfanthen) [inline src/assets/data/edition/series/2/section/2a/m37/svg-sheets.json:10]:
```suggestion
                        "svg": "assets/img/edition/series/2/section/2a/m37/M37_Textfassung1-1von1-final.svg",
```

File: src/assets/data/edition/series/2/section/2a/m37/svg-sheets.json [hunk 2]
```diff
@@ -0,0 +1,63 @@
+{
+    "sheets": {
+        "workEditions": [],
+        "textEditions": [
+            {
+                "id": "M_37",
+                "label": "M 37",
```
Comment (musicEnfanthen) [inline src/assets/data/edition/series/2/section/2a/m37/svg-sheets.json:7]:
```suggestion
                "label": "M 37 einzige Textfassung",
```

File: src/assets/data/edition/series/2/section/2a/m37/svg-sheets.json [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+        ],
+        "sketchEditions": [
+            {
+                "id": "M_37_Sk1",
+                "label": "M 37 Sk1",
+                "content": [
+                    {
+                        "svg": "assets/img/edition/series/2/section/2a/m37/M37_Sk1-1von1-final.svg",
+                        "image": "",
+                        "partial": "a"
```
Comment (musicEnfanthen) [inline src/assets/data/edition/series/2/section/2a/m37/svg-sheets.json:26]:
```suggestion
                        "image": ""
```

File: src/assets/data/edition/series/2/section/2a/m37/svg-sheets.json [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+        "workEditions": [],
+        "textEditions": [
+            {
+                "id": "M_37",
+                "label": "M 37",
+                "content": [
+                    {
+                        "svg": "assets/img/edition/series/2/section/2a/m37/M37_Textfassung1-1von2-final.svg",
+                        "image": "",
+                        "partial": "a"
```
Comment (musicEnfanthen) [inline src/assets/data/edition/series/2/section/2a/m37/svg-sheets.json:12]:
```suggestion
                        "image": ""
```

File: src/assets/data/edition/series/2/section/2a/m37/svg-sheets.json [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+                ],
+                "convolute": "A"
+            },
+            {
+                "id": "M_37_Sk3",
+                "label": "M 37 Sk3",
+                "content": [
+                    {
+                        "svg": "assets/img/edition/series/2/section/2a/m37/M37_Sk3-1von3-final.svg",
+                        "image": ""
```
Comment (musicEnfanthen) [inline src/assets/data/edition/series/2/section/2a/m37/svg-sheets.json:48]:
```suggestion
                        "image": "",
                        "partial": "a"
```

File: src/assets/data/edition/series/2/section/2a/m37/svg-sheets.json [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+                    },
+                    {
+                        "svg": "assets/img/edition/series/2/section/2a/m37/M37_Sk3-2von3-final.svg.svg",
+                        "image": ""
```
Comment (musicEnfanthen) [inline src/assets/data/edition/series/2/section/2a/m37/svg-sheets.json:52]:
```suggestion
                        "image": "",
                        "partial": "b"
```

File: src/assets/data/edition/series/2/section/2a/m37/svg-sheets.json [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+                    },
+                    {
+                        "svg": "assets/img/edition/series/2/section/2a/m37/M37_Sk3-1von3-final.svg",
+                        "image": ""
```
Comment (musicEnfanthen) [inline src/assets/data/edition/series/2/section/2a/m37/svg-sheets.json:56]:
```suggestion
                        "image": "",
                        "partial": "c"
```

File: src/assets/data/edition/series/2/section/2a/m37/svg-sheets.json [hunk 8]
```diff
@@ -0,0 +1,63 @@
...
+            {
+                "id": "M_37_Sk3",
+                "label": "M 37 Sk3",
+                "content": [
+                    {
+                        "svg": "assets/img/edition/series/2/section/2a/m37/M37_Sk3-1von3-final.svg",
+                        "image": ""
+                    },
+                    {
+                        "svg": "assets/img/edition/series/2/section/2a/m37/M37_Sk3-2von3-final.svg.svg",
```
Comment (musicEnfanthen) [inline src/assets/data/edition/series/2/section/2a/m37/svg-sheets.json:51]:
```suggestion
                        "svg": "assets/img/edition/series/2/section/2a/m37/M37_Sk3-2von3-final.svg",
```

File: src/assets/data/edition/series/2/section/2a/m37/svg-sheets.json [hunk 9]
```diff
@@ -0,0 +1,63 @@
+{
+    "sheets": {
+        "workEditions": [],
+        "textEditions": [
+            {
+                "id": "M_37",
```
Comment (musicEnfanthen) [inline src/assets/data/edition/series/2/section/2a/m37/svg-sheets.json:6]:
```suggestion
                "id": "M_37_TF1",
```

File: src/assets/data/edition/series/2/section/2a/M 37/textcritics.json [hunk 10]
```diff
@@ -0,0 +1,199 @@
...
+                    "svgGroupId": "g22161",
+                    "measure": "20",
+                    "system": "13",
+                    "position": "2/4",
+                    "comment": "Viertelnote überschreibt Halbe Note."
+                }
+            ]
+        },
+        {
+            "id": "M_37_TF",
```
Comment (musicEnfanthen) [inline src/assets/data/edition/series/2/section/2a/M 37/textcritics.json:162]:
```suggestion
            "id": "M_37_TF1",
```

File: src/assets/data/edition/series/2/section/2a/M 37/textcritics.json
Comment (musicEnfanthen) [inline src/assets/data/edition/series/2/section/2a/M 37/textcritics.json:1]: Move files into folder: M 37 --> m37

File: src/assets/data/edition/series/2/section/2a/M 37/source-list.json
Comment (musicEnfanthen) [inline src/assets/data/edition/series/2/section/2a/M 37/source-list.json:1]: Move files into folder: M 37 --> m37

File: src/assets/data/edition/series/2/section/2a/M 37/textcritics.json
Comment (chael-mi) [reply src/assets/data/edition/series/2/section/2a/M 37/textcritics.json:1]: Done

File: src/assets/data/edition/series/2/section/2a/m37/source-description.json [hunk 11]
```diff
@@ -0,0 +1,952 @@
...
+                                        {
+                                            "system": "13a–14a",
+                                            "measure": "6B–8B",
+                                            "linkTo": ""
+                                        }
+                                    ],
+                                    [
+                                        {
+                                            "system": "16–17",
+                                            "measure": "{9A–11A(1–2/4), 11B(2–4/4), 12A–13A}",
```
Comment (musicEnfanthen) [inline src/assets/data/edition/series/2/section/2a/m37/source-description.json:395]:
```suggestion
                                            "measure": "{9A–13A}",
```

**Pull request merged**